Python/pytime.c

```diff
@@ -635,6 +635,16 @@ _PyTime_AsNanosecondsObject(_PyTime_t t)
     return PyLong_FromLongLong((long long)ns);
 }
 
+_PyTime_t
+_PyTime_FromSecondsDouble(double seconds)
```

Comment:
I would prefer to pass rounding mode as a function argument, rather than hardcode the rounding to _PyTime_ROUND_FLOOR.

Reply:
@vstinner 
Fixed.